src/analyses/libraryFunctions.ml

```diff
@@ -543,6 +569,183 @@ let invalidate_actions = [
     "sema_init", readsAll;
     "down_trylock", readsAll;
     "up", readsAll;
+    "acos", readsAll;
+    "acosf", readsAll;
```

Comment:
Is there a reason `acosf` and all the other type-suffixed variants of these functions aren't defined as special functions above, along with normal, e.g. `acos`?

Reply:
has been added now